### ISSUE TITLE
Vagrant build error using docker to build instance locally

### ISSUE THREAD

**Issue by nadrojisk** [splunk/attack_range#794]
I am getting an issue trying to run the attack range locally while using docker. I am running it on a Ubuntu VM using VMware on Windows 10. When I try to install vagrant within the docker container it works, but then complains virtualbox is not installed. Virtualbox is installed and located within the path on my Ubuntu VM.

- Docker version 23.0.1, build a5ee5b1
- Ubuntu Version: VERSION_ID="22.04"

```
general:
  cloud_provider: local
  attack_range_password: toor
local: {}
windows_servers:
- hostname: ar-win-dc
  windows_image: windows-2016-v3-0-0
  create_domain: '1'
  install_red_team_tools: '1'
  bad_blood: '1'
```

```
(attack-range-py3.8) root@778c0d36fdfa:/attack_range# python attack_range.py build
                   
                              __
                            .d$$b
                          .' TO$;\
                         /  : TP._;
                        / _.;  :Tb|
                       /   /   ;j$j
                   _.-"       d$$$$
                 .' ..       d$$$$;
                /  /P'      d$$$$P. |\
               /   "      .d$$$P' |\^"l
             .'           `T$P^"""""  :
         ._.'      _.'                ;
      `-.-".-'-' ._.       _.-"    .-"
    `.-" _____  ._              .-"
   -(.g$$$$$$$b.              .'
     ""^^T$$$P^)            .(:
       _/  -"  /.'         /:/;
    ._.'-'`-'  ")/         /;/;
 `-.-"..--""   " /         /  ;
.-" ..--""        -'          :
..--""--.-"         (\      .-(\
  ..--""              `-\(\/;`
    _.                      :
                            ;`-
                           :\
                           ;  bug

By: Splunk Threat Research Team [STRT] - research@splunk.com

    
The Vagrant executable cannot be found. Please check if it is in the system path.
2023-05-04 17:50:08,319 - INFO - attack_range - [action] > build

2023-05-04 17:50:08,326 - ERROR - attack_range - vagrant failed to build

```

**Comment by P4T12ICK** [comment]
When you want to use the local version of Attack Range, you need to install it directly on your physical machine as described here:
https://attack-range.readthedocs.io/en/latest/Attack_Range_Local.html

Nested virtualization is not supported.

**Comment by nadrojisk** [comment]
Their own release notes state that the local version was merged back into the attack_range repo with v3.0.
https://www.splunk.com/en_us/blog/security/attack-range-v3-0.html
![image](https://github.com/splunk/attack_range/assets/37343577/3d7c5069-9392-405f-9038-fcb4108ae6a6)

**Comment by P4T12ICK** [comment]
Yes we integrated the python code back to the Attack Range project but still we can't support this kind of nested virtualization environments. I would suggest you to use WSL on your Windows 10 computer and install Virtualbox.
